s3_upload/utils/upload.py

```
@@ -182,7 +182,7 @@ def check_buckets_exist(buckets, slack_alert_webhook=None) -> List[dict]:
 
 def upload_single_file(
     s3_client, bucket, remote_path, local_file, parent_path
-):
+) -> Tuple[str, str]:
```

Ruff (0.8.0)

Use `tuple` instead of `Tuple` for type annotation

Replace with `tuple`

(UP006)